workflow/rules/predictions.smk

```diff
@@ -1,11 +1,18 @@
 
-rule write_feature_table: # write feature table to output directory for future reference
-    input:
-        feature_table_file = lambda wildcards: get_feature_table_file(wildcards.biosample)
-    output:
-        feature_table_here = os.path.join(RESULTS_DIR, "{biosample}", "feature_table.tsv")
-    run:
-        shell("cat {input.feature_table_file} > {output.feature_table_here}")
+rule make_biosample_feature_table:  # make feature table per biosample
+	input:
+		config["ABC_BIOSAMPLES"]
+	output:
+		biosample_features = os.path.join(RESULTS_DIR, "{biosample}", "feature_table.tsv")
+	params:
+		biosample_config = config["ABC_BIOSAMPLES_MODELS"],
```

Comment:
It seems like you really just need the model directories. You can just leverage the existing BIOSAMPLES_DF to get that. Something like 
```
model_dirs = BIOSAMPLE_DF[BIOSAMPLE_DF['biosample'] == biosample]['model_dir'].to_list()
```
I believe snakemake and R should handle model_dirs being a list, but not completely sure. 

Then this way, you don't need to create a separate config file in Snakemake. (It also looks like you may've forgotten to create the ABC_BIOSAMPLES_MODELS config in the latest commit?)